cosmos.gemspec

```diff
@@ -76,22 +76,22 @@ spec = Gem::Specification.new do |s|
   s.required_ruby_version = '~> 2.2'
 
   # Runtime Dependencies
-  s.add_runtime_dependency 'bundler', '~> 1.3'
-  s.add_runtime_dependency 'rdoc', '>= 3', '< 5'
-  s.add_runtime_dependency 'rake', '>= 0', '< 13'
-  s.add_runtime_dependency 'json', '>= 1.5', '< 1.9'
-  s.add_runtime_dependency 'pry', '>= 0.9', '< 0.11'
-  s.add_runtime_dependency 'pry-doc', '>= 0.5', '< 0.7'
-  s.add_runtime_dependency 'yard', '>= 0.8', '< 0.10'
+  s.add_runtime_dependency 'bundler', '~> 1.16'
+  s.add_runtime_dependency 'rdoc', '>= 4', '< 6'
+  s.add_runtime_dependency 'rake', '~> 12.0'
+  s.add_runtime_dependency 'json', '~> 2.0'
+  s.add_runtime_dependency 'pry', '~> 0.11'
+  s.add_runtime_dependency 'pry-doc', '~> 0.11'
+  s.add_runtime_dependency 'yard', '~> 0.9'
```

Comment:
Did you up all the required versions for a reason?   This will make upgrading much harder for users in offline areas.